### PR TITLE
Storybook Menu Organism

### PR DESCRIPTION
**Pull request by FIZLIN**

### CODE REVIEW

File: packages/ui-kit-storybook/src/organisms/Menu/Menu.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import {ComponentMeta, Story} from '@storybook/react';
+import {useState} from 'react';
+
+import Menu, {MenuItemInterface, MenuPropsInterface} from './Menu';
+
+export default {
+  title: 'Organisms/Menu',
+  component: Menu
```
Comment (PAKovalev) [inline packages/ui-kit-storybook/src/organisms/Menu/Menu.stories.tsx]: Let's render menu in bottom. There is `layout` prop for this.

File: packages/ui-kit-storybook/src/organisms/Menu/Menu.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  onMenuItemSelection
+}) => {
+  const [sidePadding, setSidePadding] = useState<number>(0);
+  const [leftBlankCount, setLeftBlankCount] = useState<number>(1);
+  const [rightBlankCount, setRightBlankCount] = useState<number>(1);
+
+  const ref = useRef<HTMLDivElement>(null);
+
+  const [windowWidth, setWindowWidth] = useState(window.innerWidth);
+  useResize(ref, () => setWindowWidth(window.innerWidth));
```
Comment (PAKovalev) [inline packages/ui-kit-storybook/src/organisms/Menu/Menu.tsx:58]: After resizing count of blank hexagons is not changed. Will send you a screenshot in DM.

File: packages/ui-kit-storybook/src/organisms/Menu/Menu.tsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+  imageSrc?: string;
+  iconName?: IconNameType;
+  tooltip?: string;
+}
+
+export interface MenuPropsInterface {
+  leftActions: MenuItemInterface[];
+  centerActions: MenuItemInterface[];
+  rightActions: MenuItemInterface[];
+
```
Comment (PAKovalev) [inline packages/ui-kit-storybook/src/organisms/Menu/Menu.tsx:43]: We must have submenu prop which appears after clicking on center hexagon.

**Review by PAKovalev — CHANGES_REQUESTED** [review]

**Review by PAKovalev — APPROVED** [review]

File: packages/ui-kit-storybook/src/organisms/Menu/Menu.tsx [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+const Menu: FC<MenuPropsInterface> = ({
+  leftActions,
+  centerActions,
+  rightActions,
+  activeMenuItemKey,
+  onMenuItemSelection
+}) => {
+  const [sidePadding, setSidePadding] = useState<number>(0);
+  const [leftBlankCount, setLeftBlankCount] = useState<number>(1);
+  const [rightBlankCount, setRightBlankCount] = useState<number>(1);
```
Comment (dmitry-yudakov) [inline packages/ui-kit-storybook/src/organisms/Menu/Menu.tsx]: Maybe `const [sidePadding, leftBlankCount, rightBlankCount] = useMemo(() => {...},[...])` would be a better alternative to `useState` + `useEffect`. The logic could also be moved to separate function to make the component logic a bit more compact.

**Review by dmitry-yudakov — APPROVED** [review]
approved with one improvement suggestion

**Pull request merged**